catalog/dags/common/elasticsearch.py

```diff
@@ -143,6 +143,7 @@ def trigger_and_wait_for_reindex(
     max_docs: int | None = None,
     refresh: bool = True,
     slices: Union[int, Literal["auto"]] = "auto",
+    poke_interval: int = REFRESH_POKE_INTERVAL,
```

Comment:
I got an import error on the homepage because of the `int` type.

```python
airflow.exceptions.AirflowException: Operator arg `poke_interval` must be timedelta object or a non-negative number
```